samples/bluetooth/peripheral_fast_pair/src/bt_adv_helper.c

```diff
@@ -52,20 +52,30 @@ BT_CONN_CB_DEFINE(conn_callbacks) = {
 	.connected        = connected,
 };
 
-static void bond_count_helper(const struct bt_bond_info *info, void *user_data)
+static inline bool pairing_mode(enum bt_fast_pair_adv_mode fp_adv_mode)
```

Comment:
using inline has no practical sense, does it?
declaring the function static will make the compiler do right thing with it anyway.

Reply:
Ok, removed. I initially wanted to provide compiler with a hint here.